src/azure-cli/azure/cli/command_modules/acs/tests/latest/recordings/test_aks_approuting_update_with_monitoring_addon_enabled.yaml

```diff
@@ -0,0 +1,2432 @@
+interactions:
```

Comment:
As you've mark the corresponding test case as live_only(), it's not necessary to commit this recording file

Reply:
removed